cmd/console/deploy/deploy.go

```diff
@@ -41,25 +42,26 @@ func NewDeployCmd() *cobra.Command {
 				return cmd.MarkFlagRequired("project")
 			}
 
-			// set the flag only in case it is defined
+			// set these flag only in case they are defined
 			skipCertificate, _ = cmd.Flags().GetBool("insecure")
+			certificatePath = viper.GetString("ca-cert")
 
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
 			f, err := factory.FromContext(cmd.Context(), sdk.Options{
-				APIBaseURL:      baseURL,
-				APIToken:        apiToken,
-				SkipCertificate: skipCertificate,
+				APIBaseURL:            baseURL,
+				APIToken:              apiToken,
+				SkipCertificate:       skipCertificate,
+				AdditionalCertificate: certificatePath,
 			})
 			if err != nil {
 				return err
 			}
 
 			deployData, err := f.MiaClient.Deploy.Trigger(projectId, cfg)
 			if err != nil {
-				f.Renderer.Error(err).Render()
-				return nil
+				return err
```

Comment:
Why it is not used the renderer?

Reply:
Because using just the renderer prevents the command to exit with a status code different from 0.

The error is then printed to std out by these lines: https://github.com/mia-platform/miactl/blob/feature/select-custom-certificate/cmd/root.go#L53-L56 which are in charge also to select correct status code.

**Note:** returning an error is not a solution since the error message would be printed three times:
- from the renderer
- from cobra error handler (disabled setting `SilenceErrors: true` in root command config)
- from miactl custom error handler